packages/apollo-server-core/src/__tests__/landingPageConfigHTML.test.ts

```diff
@@ -0,0 +1,158 @@
+import {
```

Comment:
Can we move this file to `apollo-server-core/src/plugin/landingPage/__tests__`?

There's a bit of additional config required to make TS happy.
* You'll need to add an entry to `/tsconfig.test.json` (see `cacheControl` and `usageReporting` plugin entries)
* You'll need to add a `apollo-server-core/src/plugin/landingPage/__tests__/tsconfig.json` (this can be copied verbatim from `apollo-server-core/src/plugin/cacheControl/__tests__/tsconfig.json`)

And maybe rename the file to `getEmbeddedExplorerHTML.test.ts` since that's a bit more precisely what we're testing in this file.